src/tests/ftest/pool/attribute.py

```diff
@@ -198,13 +200,13 @@ def test_pool_attribute_asyn(self):
 
         self.add_pool()
         expected_for_param = []
-        name = self.params.get("name", '/run/attrtests/name_handles/*/')
+        name = self.params.get("name", '/run/attrtests/test_handles/*/')
         # workaround until async functions are fixed
         if name[0] is not None and "Negative" in name[0]:
             pass
         else:
             expected_for_param.append(name[1])
-        value = self.params.get("value", '/run/attrtests/value_handles/*/')
+        value = self.params.get("value", '/run/attrtests/test_handles/*/')
```

Comment:
This should not be changed - see comment in yaml file.

Reply:
Fixed